Resources/ServerInfo/Guidebook/Science/Cyborgs.xml

```diff
@@ -18,28 +18,28 @@
   </Box>
   Both brains can be fabricated without requiring any additional research.
 
-  ## Chassis
-  While all cyborgs share the same endoskeleton, not all share the same chassis. The chassis determines what modules the cyborg can have, along with the [color=#a4885c]departmental radio channel[/color] they correspond to. By default, they will always have access to [color=#D381C9]Science[/color] and [color=green]station-wide[/color] frequencies, along with having [color=#a4885c]all-access[/color].
+  ## Borg types
+  Once created, a borg needs to specialize its chassis to a duty on the station. This determines what modules it starts with, which additional modules can be installed, and what [color=#a4885c]departmental radio channel[/color] they have access to. All borgs have access to the [color=#D381C9]Science[/color] and [color=green]station-wide[/color] radio. All borgs have [color=#a4885c]all-access[/color].
```

Comment:
```suggestion
  Once created, a cyborg needs to specialize its chassis to fulfill its duties on the station. This determines what modules it starts with, which additional modules can be installed, and what [color=#a4885c]departmental radio channel[/color] it has access to. All cyborgs have access to the [color=#D381C9]Science[/color] and [color=green]station-wide[/color] radio channels. All cyborg types have [color=#a4885c]all-access[/color].
```

Reply:
I didn't apply the diff about "fulfill its duties" because it doesn't cover what I tried to say.

The sentence says "specialize its chassis to a duty (noun)", not "specialize its chassis to do a duty (verb)". This is a minor but important difference.